### DISCUSSION TITLE
Why VuePress 2.x?

### DISCUSSION THREAD

**Discussion by dagilleland** [vuepress/core#247] [Q&A] [answered]
It might seem a silly question, or one already clear to the inner community, but I (and others, probably) are wondering **why do we need a new version of VuePress?**  🤔 

It is just because of Vue 3.x? Are you wanting VuePress to be/do something else? Is there something that needs to be fundamentally changed (breaking-change-like) or major enhancement/feature?

(The question is not to 'dis on VuePress. I love it! I've been using it for years. But after a while, it became a "setup-forget", and then I wandered back to discover plans for the new version.)

**Comment by dagilleland** [comment, marked as answer]
And... to answer my own (and other's) questions: You can find some of the "why" in the [**Migration from V1.x**](https://v2.vuepress.vuejs.org/guide/migration.html#migrating-from-v1) guide.

**Comment by Mister-Hope** [comment]
Other possible goals:

- dev and build Speed
- application performance improvement
- fix of hmr
- bump deps to be used in monorepos with newer deps without breaking the whole project

E.g.: For me, I cannot use vuepress v1 in my webpack 5 monorepo poject because it will break the deps somewhere because all the webpack plugins are almost in different major versions.
